cassandra/io/asyncioreactor.py

```diff
@@ -88,9 +90,11 @@ def __init__(self, *args, **kwargs):
 
         self._connect_socket()
         self._socket.setblocking(0)
-
-        self._write_queue = asyncio.Queue()
-        self._write_queue_lock = asyncio.Lock()
+        loop_args = dict()
+        if sys.version_info[0] == 3 and sys.version_info[1] < 10:
+            loop_args['loop'] = self._loop
+        self._write_queue = asyncio.Queue(**loop_args)
+        self._write_queue_lock = asyncio.Lock(**loop_args)
```

Comment:
This came up when we were working on [PYTHON-1313](https://datastax-oss.atlassian.net/browse/PYTHON-1313).  [Docs](https://docs.python.org/3/whatsnew/3.10.html#removed) I found at the time indicated that since Python 3.7 the event loop had been derived from the current threads event loop and the "loop" param was essentially ignored.  We're currently looking at 3.8 through 3.12 so this block only applies to Python 3.8.x and 3.9.x.  Do you think it's worth it to continue to pass a "loop" param for those cases?

Reply:
Yes it seems to be needed, I.e. taking it down got it broke.

Reply:
+1